test/e2e/test-app/fixtures/javascript main with spaces.js

```diff
@@ -0,0 +1,3 @@
+if (process.type === 'browser') {
+  throw new Error('Error triggered in renderer process');
```

Comment:
`process.type === 'browser'` indicates the main process.

Reply:
Yeah crazy isn't it. I think the decision was made early on and now they can't change it.

Reply:
But yeah, the error text are wrong...